revolv/administrator/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from revolv.payments.models import AdminAdjustment
+from revolv.project.models import Project
+from django.forms.models import modelform_factory
+
+
+AdjustmentForm = modelform_factory(AdminAdjustment)
```

Comment:
You should add `exclude=[]` so the logger doesn't complain at us about being removed in django 1.8